g2p/mappings/langs/oka/oka_equiv.csv

```diff
@@ -0,0 +1,15 @@
+’,'
+‘,'
```

Comment:
would be nice to have a standard list of confusable apostrophes here that we could just extend from

Reply:
That's true, we keep having to recreate this list!